src/wmtk/EdgeMesh.cpp

```diff
@@ -0,0 +1,86 @@
+#include "EdgeMesh.hpp"
+namespace wmtk {
+Tuple EdgeMesh::vertex_tuple_from_id(long id) const
+{
+    throw("this function has not been implemented! -- EdgeMesh.hpp/EdgeMesh.cpp");
+    return Tuple(-1, -1, -1, id, get_cell_hash_slow(id));
+}
+
+EdgeMesh::EdgeMesh()
+    : Mesh(0)
+{}
+
+
+EdgeMesh::EdgeMesh(long size)
+    : EdgeMesh()
+{
+    initialize(size);
+}
+
+Tuple EdgeMesh::switch_tuple(const Tuple& tuple, PrimitiveType type) const
+{
+    throw("this function has not been implemented! -- EdgeMesh.hpp/EdgeMesh.cpp");
+    throw std::runtime_error("Tuple switch: Invalid primitive type");
+    return tuple;
+}
+bool EdgeMesh::is_ccw(const Tuple&) const
+{
+    throw("this function has not been implemented! -- EdgeMesh.hpp/EdgeMesh.cpp");
+    // trivial orientation so nothing can happen
+    return true;
+}
+bool EdgeMesh::is_boundary(const Tuple&) const
+{
+    throw("this function has not been implemented! -- EdgeMesh.hpp/EdgeMesh.cpp");
+    // every point is on the interior as it has no boundary simplices
+    return false;
+}
+
+bool EdgeMesh::is_boundary_vertex(const Tuple&) const
+{
+    throw("this function has not been implemented! -- EdgeMesh.hpp/EdgeMesh.cpp");
+    // every point is on the interior as it has no boundary simplices
+    return false;
+}
+
+void EdgeMesh::initialize(long count)
+{
+    throw("this function has not been implemented! -- EdgeMesh.hpp/EdgeMesh.cpp");
+    set_capacities({count});
```

Comment:
ah i suspect you borrowed your initialization from PointMesh - for EdgeMesh initialization needs to be a collection of pairs of indices (the endpoints of each edge). The capacities should be set as `{number of vertices, number of edges}`